src/presenter/points-presenter.js

```diff
@@ -0,0 +1,107 @@
+import { render, replace } from '../framework/render.js';
+import NewSortView from '../view/sort-view.js';
+import ContainerPointsView from '../view/container-points-view.js';
+import FiltersTitleView from '../view/filters-title-view.js';
+import { getDifferencesDates } from '../utils.js';
+import PointPresenter from './point-presenter.js';
+
+
+const tripFilters = document.querySelector('.trip-controls__filters');
+
+export default class PointsPresenter {
+  containerPointsView = new ContainerPointsView();
+  sort = new NewSortView();
+  filtersTitle = new FiltersTitleView();
+  activeFilter = 'everything';
+  activeSort = 'sort-day';
+  #points = new Set();
+
+  constructor({pointsContainer, pointsModel}) {
+    this.pointsContainer = pointsContainer;
+    this.pointsModel = pointsModel;
+  }
+
+  deleteAllPoints = () => {
+    document.querySelectorAll('.trip-events__item').forEach((point) => point.remove());
+    document.querySelectorAll('.event--edit').forEach((editPoint) => editPoint.remove());
+  };
+
+  get points() {
+    const points = this.pointsModel.getPoints();
+    return points;
+  }
+
+  #renderPoint(point, boardOffers, boardDestinations) {
+    const pointPresenter = new PointPresenter(point, boardOffers, boardDestinations, this.containerPointsView);
+    pointPresenter.init();
+    this.#points.add(pointPresenter);
+  }
+
+  #renderPoints(boardOffers, boardDestinations) {
+    this.points.forEach((point) => this.#renderPoint(point, boardOffers, boardDestinations));
+  }
+
+  #redrawPoints = (boardPoints) => {
```

Comment:
#redrawPoints() {
  this.#points.forEach((pointPresenter) => {
    pointPresenter.init();
  });
}
```